db/migrate/20151026152536_remove_automate_worker.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveAutomateWorker < ActiveRecord::Migration
+  class MiqWorker < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def change
+    MiqWorker.destroy_all(:type => 'MiqAutomateWorker')
```

Comment:
Agreed, delete_all would be better here.

Also, like other data migrations wrap in `say_with_time`